src/http/mod.rs

```diff
@@ -1,7 +1,9 @@
 pub(self) mod cf;
 pub(crate) mod feature;
 pub(self) mod legacy;
+mod port;
```

Comment:
do we want this somewhere that is not couple up with our `http` module stuff? i literally would put this in utils lol. could also just make top level port.rs but that feels messy